civiform/civiform
# Revise flow for adding/removing program admins

### Description

Revise flow for adding/removing program admins via email address.

## Release notes

Revise flow for adding/removing program admins via email address. The new flow adds or deletes a single admin per action and uses standard UI elements.

### Checklist

#### General

Read the full guidelines for PRs [here](https://docs.civiform.us/contributor-guide/developer-guide/technical-contribution-guide#guidelines-for-pull-requests)


### Instructions for manual testing

- Navigate to Program Dashboard
- Under three-dot menu, choose "Manage Program Admins"
- Add and delete admins. Accounts must exist and the accounts cannot be CiviForm Admins
- Click "Back" to return to Program Dashboad
- Under three-dot menu, choose "Manage Program Admins" to ensure the list has been properly persisted.

### Issue(s) this completes

Fixes #4876

![image](https://github.com/civiform/civiform/assets/1391583/d88ea568-78bb-44dd-a10f-a9d4d1218e7a)
